stories/modules-tagmanager-setup.stories.js

```diff
@@ -162,6 +184,44 @@ storiesOf( 'Tag Manager Module/Setup/Secondary AMP', module )
 		registry.dispatch( STORE_NAME ).setAMPContainerID( ampContainer.publicId );
 		registry.dispatch( STORE_NAME ).setInternalAMPContainerID( ampContainer.containerId );
 
+		return <Setup registry={ registry } />;
+	} )
+	.add( 'Singular property ID, Analytics inactive', ( registry ) => {
+		registry.dispatch( MODULES_ANALYTICS ).receiveGetSettings( { propertyID: '' } );
+		const webContainerVersion = fixtures.liveContainerVersions.web.gaWithVariable;
+		const ampContainerVersion = fixtures.liveContainerVersions.amp.ga;
+		const accountID = webContainerVersion.accountId;
+		registry.dispatch( STORE_NAME ).receiveGetAccounts( fixtures.accounts );
+		registry.dispatch( STORE_NAME ).receiveGetContainers( [
+			webContainerVersion.container,
+			ampContainerVersion.container,
+		], { accountID } );
+		parseLiveContainerVersionIDs( webContainerVersion, ( { internalContainerID } ) => {
+			registry.dispatch( STORE_NAME ).receiveGetLiveContainerVersion( webContainerVersion, { accountID, internalContainerID } );
+		} );
+		parseLiveContainerVersionIDs( ampContainerVersion, ( { internalContainerID } ) => {
+			registry.dispatch( STORE_NAME ).receiveGetLiveContainerVersion( ampContainerVersion, { accountID, internalContainerID } );
+		} );
+
+		return <Setup registry={ registry } />;
+	} )
+	.add( 'Multiple property IDs, Analytics inactive', ( registry ) => {
+		registry.dispatch( MODULES_ANALYTICS ).receiveGetSettings( { propertyID: '' } );
+		const webContainerVersion = fixtures.liveContainerVersions.web.gaWithVariable;
+		const ampContainerVersion = fixtures.liveContainerVersions.amp.noGA;
+		const accountID = webContainerVersion.accountId;
+		registry.dispatch( STORE_NAME ).receiveGetAccounts( fixtures.accounts );
+		registry.dispatch( STORE_NAME ).receiveGetContainers( [
+			webContainerVersion.container,
+			ampContainerVersion.container,
+		], { accountID } );
+		parseLiveContainerVersionIDs( webContainerVersion, ( { internalContainerID } ) => {
+			registry.dispatch( STORE_NAME ).receiveGetLiveContainerVersion( webContainerVersion, { accountID, internalContainerID } );
+		} );
+		parseLiveContainerVersionIDs( ampContainerVersion, ( { internalContainerID } ) => {
+			registry.dispatch( STORE_NAME ).receiveGetLiveContainerVersion( ampContainerVersion, { accountID, internalContainerID } );
+		} );
+
```

Comment:
I think it would be good to add full Storybook coverage here, also for e.g. when there is an Analytics property in GTM and Analytics is already active.

We should probably also add the same coverage for AMP and non-AMP, currently you only cover primary and secondary AMP for the new Storybook items.